app/Event/EventReadServiceProvider.php

```diff
@@ -52,5 +59,23 @@ function () use ($container): JSONLDMainLanguageQuery {
                 );
             }
         );
+
+        $container->addShared(
+            EventLocationHistoryProjector::class,
```

Comment:
If you look at other projectors they all use `Place` in their name, for example `PlaceRelationsProjector`. If you think about it, this is the `PlaceRelationsHistoryProjector`

Reply:
I would keep `event` in the name because is the relation between `Event` and `Place` that is being tracked.
It also listens to Event related commands like `EventCreated`, not place related events.
Can we land on "EventRelationsPlaceHistoryProjector"?

I put Event first because it is the relation from the viewpoint of an event we track.

Reply:
`EventRelationsPlaceHistoryProjector` is quiet long, maybe `EventPlaceHistoryProjector`?